### ISSUE TITLE
QdtViz Barchart tooltip

### ISSUE THREAD

**Issue by fawazeez** [qlik-demo-team/qdt-components#215]
I am using QdtViz for a barchart using ID.  When I have multiple dimension the tool tip is showing some functions as in image attached. I am using a Feb 2019 Release Qlik sense. I have tried using the same chart on simple mashup in Dev hub and is working fine

![image](https://user-images.githubusercontent.com/9716898/74730767-2dfb3f80-5260-11ea-8afe-6df1d9e1f407.png)

**Comment by fawazeez** [comment]
@fkabinoff  @yianni-ververis  Sorry guys I am not sure if this is the right way to seek help. Can you please help me on this issue

**Comment by yianni-ververis** [comment]
This seems to be a Qlik Sense issue from the Capability API since all QdtComponent does is use the getObject method.

TBH I have never seen that before

**Comment by fawazeez** [comment]
Yeah I checked the same objectid via mashup editor and its working fine. I have problem only when using qdt and that function is displayed when i have more than one dimension. Can you advice how can i confirm or is there any chance of version issues
![image](https://user-images.githubusercontent.com/9716898/75367856-5aefc800-58da-11ea-8304-c4343415637b.png)

**Comment by yianni-ververis** [comment]
Take that mashup from the editor and run it on a different server

**Comment by fawazeez** [comment]
i ran mashup on my laptop and its working good except for the styling
![image](https://user-images.githubusercontent.com/9716898/75752440-26e52e80-5d42-11ea-96f0-1278581d0cbb.png)

Is there any chance of version issues between Qdt and our Qlik

**Comment by fawazeez** [comment]
or I think it could be any of the package on our internal application conflicting
